lighthouse-core/report/html/renderer/util.js

```diff
@@ -81,6 +81,7 @@ class Util {
         // into 'debugdata' (LHR ≥5.0).
         // @ts-expect-error tsc rightly flags that these values shouldn't occur.
         if (audit.details.type === undefined || audit.details.type === 'diagnostic') {
+          // @ts-expect-error
```

Comment:
this one makes less sense to me than the one immediately above it, what's the reason?

Reply:
> this one makes less sense to me than the one immediately above it, what's the reason?

it's because the conditional above only lets in `audit.details` that shouldn't be possible, so it's of type `never` and can't be assigned to. Definitely worth a `@ts-expect-error` comment for that, though.